src/QirRuntime/lib/QIR/quantum__rt.hpp

```diff
@@ -91,13 +90,13 @@ extern "C"
     // ------------------------------------------------------------------------
 
     // Allocates space for a tuple requiring the given number of bytes and sets the reference count to 1.
-    QIR_SHARED_API QirTupleHeader* quantum__rt__tuple_create(int64_t); // NOLINT
+    QIR_SHARED_API char* quantum__rt__tuple_create(int64_t); // NOLINT
 
     // Indicates that a new reference has been added.
-    QIR_SHARED_API void quantum__rt__tuple_reference(QirTupleHeader*); // NOLINT
+    QIR_SHARED_API void quantum__rt__tuple_reference(char*); // NOLINT

```

Comment:
I would prefer "add_reference"/"remove_reference" wording to reference/unreference. Just "reference" is unclear, since it is both a verb and a noun. Yes, I realize this is not directly part of this checkin.

Reply:
The naming is inherited from the QIR spec. Personally, I'd prefer addref/release terminology as I'm used to it from COM era, but here I believe it's more important to be consistent with the naming of the QIR runtime methods.